SCAutolib/models/openssl_conf.py

```diff
@@ -0,0 +1,69 @@
+"""
+This module provide information and methods to create and modify
+openssl cnf file. Following methods are implemented:
+
+create: create content of internal file object representing openssl cnf file
+save:  save content of internal file object representing openssl cnf file to
+       .cnf file specified by user
+"""
+from pathlib import Path
+from typing import Union
+
+from SCAutolib import TEMPLATES_DIR
+from SCAutolib.models.file import File
+
+
+class OpensslCnf(File):
```

Comment:
We should probably move all the File related classes under one file.py file

Reply:
> We should probably move all the File related classes under one file.py file

I though about it and did not know what is better and why so I left it this way. Are there any pros and cons of having it in one file?

Reply:
I can only think of pros:
1. All file related classes are under one file.
2. The other classes inherit from File and it would be easier to check the base class for reference when trying to figure things out.
3. Folder structure is less cluttered.

Reply:
Ack, will do.